### DISCUSSION TITLE
"CURSO Fundamentos do C#" ainda vale a pena visto que é no .net 3.1?

### DISCUSSION THREAD

**Discussion by MatheusAlbane** [balta-io/forum#390] [Tira Dúvidas] [answered]
Boa notie!
Estou fazendo o curso "Fundamentos do C#" do balta, porém, vi que esta "defasado" em relação a versão atual do .net.
Ainda vale a pena fazê-lo? Se não, qual sugestão?

Obrigado!

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @MatheusAlbane, não está defasado, os conceitos são os mesmos, o que mudou foi a versão do Framework que está atualizada nos cursos das carreiras.
